.github/workflows/main.yml

```diff
@@ -48,10 +49,12 @@ jobs:
           - Moose64-10
           - Moose64-9.0
           - Moose64-8.0
+          - GemStone64-3.5.3
```

Comment:
Are you sure that you want to add this GemStone version? There's probably a good reason for its absence.